app/views/pages/developer-guide/pos-marketplace-template/built-in-features.liquid

```diff
@@ -5,285 +5,285 @@ metadata:
+* I can remove an item from the shopping cart.
+* I can change the quantity of an item in the shopping cart.
+* the number of items in the shopping cart in navigation is displayed for me.
+* the price for each product and the total price for all products added to the cart are displayed for me.
 * I can continue shopping after adding an item to the cart.
 * I can't add/have more than the limit provides for maximum shopping cart price and/or max line items.
 
 As a **registered user**,
 
-* I can add a product to the shopping cart. 
-* I can add the same product to the shopping cart. 
+* I can add a product to the shopping cart.
+* I can add the same product to the shopping cart.
 * I can add another product to the shopping cart from the same Merchant.
 * I can add another product to the shopping cart from another Merchant.
 * I can remove an item from the shopping cart.
 * I can go back to the shopping cart if I abandon the flow before Payment.
-* I can change the quantity of an item in the shopping cart. 
-* I can see the number of items in the shopping cart in navigation. 
+* I can change the quantity of an item in the shopping cart.
+* the number of items in the shopping cart in navigation is displayed for me.
 * I can continue shopping after adding an item to the cart.
 * I can't add/have more than the limit provides for maximum shopping cart price and/or max line items.
 
 
-### Search 
+### Search
 
-Search functionality for items, with quick search feature, and sorting by price, relevance, merchant, category, and keyword. 
+Search functionality for items, with quick search feature, and sorting by price, relevance, merchant, category, and keyword.
 
 
-#### User stories 
+#### User stories
 
 As an **anonymous** or **registered user**,
 
-* I can see a list of relevant products when I enter a search keyword. 
+* a list of relevant products when I enter a search keyword is displayed for me.
 * I can sort search results by price (ascending and descending) and relevance.
 * I can set a minimum and maximum Price.
 * I can set a category filter and min/max price all at once.
 * I can search for products belonging to multiple merchants.
-* I can use 'quick search' for quick product search. 
-* I won't see affiliate links when sorting by price. 
-* I am taken to the product detail page after clicking a product image or name in the search results. 
+* I can use 'quick search' for quick product search.
+* I won't get affiliate links when sorting by price.
+* I am taken to the product detail page after clicking a product image or name in the search results.
 
-As a **Buyer**, 
+As a **Buyer**,
 
-* I can see SEO friendly URLs for items.
-* I can see SEO friendly search page.
+* I can have SEO friendly URLs for items.
+* I can have SEO friendly search page.
 * I can search for items by category, and keyword.
 * I can browse products belonging to a selected Merchant.
-* I can see a list of all Merchants.
+* I can explore a list of all Merchants.
```

Comment:
Those user stories are usually written in language compatible with e2e testing, and computers dont explore, they only see what we tell them what to look for.

Example test using Given/When/Then syntax: https://github.com/mdyd-dev/platformos-documentation/blob/8127d0b7f912d6d73a1bbcdffbfca21997936181/tests/codecept/Navigation_test.js#L6

Reply:
I understand @pavelloz. I updated this for inclusive language, as it's content aimed at our human visitors. Does this harm the testing process? I imagine we use a different list for that.

Reply:
As discussed, I will revert these changes and add a note that explaines why we use "see".